system/component_state_monitor/config/topics.yaml

```diff
@@ -0,0 +1,182 @@
+- module: map
+  mode: [online, planning_simulation]
+  type: launch
+  args:
+    node_name_suffix: vector_map
+    topic: /map/vector_map
+    topic_type: autoware_auto_mapping_msgs/msg/HADMapBin
+    best_effort: false
+    transient_local: true
+    warn_rate: 0.0
+    error_rate: 0.0
+    timeout: 0.0
+
+- module: map
+  mode: [online]
+  type: launch
+  args:
+    node_name_suffix: pointcloud_map
+    topic: /map/pointcloud_map
+    topic_type: sensor_msgs/msg/PointCloud2
+    best_effort: false
+    transient_local: true
+    warn_rate: 0.0
+    error_rate: 0.0
+    timeout: 0.0
+
+- module: localization
+  mode: [online, planning_simulation]
+  type: autonomous
+  args:
+    node_name_suffix: initialpose3d
+    topic: /initialpose3d
+    topic_type: geometry_msgs/msg/PoseWithCovarianceStamped
+    best_effort: false
+    transient_local: false
+    warn_rate: 0.0
+    error_rate: 0.0
+    timeout: 0.0
+
+- module: localization
+  mode: [online]
+  type: autonomous
+  args:
+    node_name_suffix: pose_twist_fusion_filter_pose
+    topic: /localization/pose_twist_fusion_filter/pose
+    topic_type: geometry_msgs/msg/PoseStamped
+    best_effort: false
+    transient_local: false
+    warn_rate: 5.0
+    error_rate: 1.0
+    timeout: 1.0
+
+- module: sensing
```

Comment:
obstacle segmentation is part of perception.

Reply:
The [original setting](https://github.com/autowarefoundation/autoware.universe/blob/main/system/ad_service_state_monitor/config/ad_service_state_monitor.param.yaml#L55-L56) is sensing. Can I change it?

Reply:
Yes. It is moved from sensing to perception during the core/universe transplant.

Reply:
Done https://github.com/autowarefoundation/autoware.universe/pull/2120/commits/fa5f10655d80a2d08e994d435bca9809217de2e5.